packages/core/src/lib/connection_manager.ts

```diff
@@ -11,6 +13,7 @@ import {
   IPeersByDiscoveryEvents,
   IRelay,
   KeepAliveOptions,
+  MultiaddrStr,
```

Comment:
there should be a type for it in `multiaddr` package

Reply:
@danisharora099, were you able to find a type in the lib or we need to intro a new one?